File: Codeforces/Fun_with_Integers.py
#https://codeforces.com/contest/1062/problem/D
import math
n = int(input())
arr = [0] * (n+1)
for i in range(2,n+1):
	count = 0
	#print("i:",i)
	for j in range(2, math.floor(i**0.5) +1):
		#print(j)
		if (i%j == 0):
			score = int(i/j)
			if score !=1:
				count += 4 * score
				if (j!=score):
					count += 4 * j

	arr[i] = arr[i-1] + count
#print(arr)
print(int(arr[-1]))

File: Codeforces/Fun_with_Integers.cpp
#include <bits/stdc++.h>
#define ll long long 
using namespace std;
ll arr[100005];
int main()
{
	int n;
	cin>>n;
	for (int i = 2;i < n+1;++i)
	{
		ll count = 0;
		for (int j = 2;j<floor(sqrt(i))+1; ++j)
		{
			if (i%j == 0)
			{
				int score = i/j;
				if (score !=1)
				{
					count+=4*score;
					if (j!=score)
					{
						count += 4 * j;
					}
				}
			}
		}
		arr[i] = arr[i-1]+count;
	}

	cout<<arr[n];
}